builder/main.py

```diff
@@ -108,8 +108,12 @@ def inspect_host(spec):
     compiler_path = toolchain.compiler_path()
     if not compiler_path:
         compiler_path = '(Will Install)'
-    print('  Compiler: {} (version: {}) {}'.format(
-        spec.compiler, toolchain.compiler_version, compiler_path))
+    if spec.compiler == 'default':
+        print('  Default Compiler resolved to: {} (version: {}) {}'.format(
+            toolchain.compiler, toolchain.compiler_version, compiler_path))
+    else:
+        print('  Compiler: {} (version: {}) {}'.format(
+            spec.compiler, toolchain.compiler_version, compiler_path))
```

Comment:
`spec.compiler` is from old code, but it can be `toolchain.compiler` in both branches. the reason for if is i wanted to have a more explicit message in cases where you pass in default in build spec and it resolved to a given compiler

Reply:
ah, i did switch it to use toolchain in both places locally, but missed file in the previous commit. latest one has it using toolchain.compiler in both